index.js

```diff
@@ -361,16 +361,44 @@ Synopsis.prototype.winner = function () {
 
 Synopsis.prototype.winners = function (n, weights) {
   const results = weights || this.topN()
-  let i, point, upper, winners
+  const pinned = []
+  let count = 0
+  let winners
 
   if (!results) return
 
   winners = []
-
   if ((typeof n !== 'number') || (n < 1)) n = 1
+
+  results.forEach((result) => {
+    let votes
+
+    if ((typeof result.pinPercentage === 'undefined') || (result.pinPercentage <= 0)) return
```

Comment:
can we do `if ((result.pinPercentage == null) || (result.pinPercentage <= 0)) return` so that we check null as well. Just in case

Reply:
+1